assets/src/apps/v2/dup.tsx

```diff
@@ -14,6 +15,13 @@ import Placeholder from "Components/v2/placeholder";
 import NormalHeader from "Components/v2/dup/normal_header";
 import NormalDepartures from "Components/v2/departures/normal_departures";
 
+const sentryDsn = document.getElementById("app")?.dataset.sentry;
```

Comment:
a way to cut down on the repeat of this init code would be to move it into its own custom hook `useSentry()` or something along those lines. 

You should be able to move all this logic into the hook definition and I believe it should work the same way.

Reply:
Good idea. Re-tested after the change and all works as expected.